src/internet_identity/src/assets.rs

```diff
@@ -73,23 +92,24 @@ pub fn init_assets() {
 // prepared only once (like injecting the canister ID).
 fn get_assets() -> [(&'static str, &'static [u8], ContentEncoding, ContentType); 8] {
     let index_html: &[u8] = INDEX_HTML_STR.as_bytes();
+    let about_html: &[u8] = ABOUT_HTML_STR.as_bytes();
```

Comment:
out of curiosity, any particular reason you created extra variables instead an hashmap (or else) of static HTML assets?

Reply:
Just for consistency; first I copy pasted `index_html` then I decided to inline it and then I reverted (for consistency with `index_html`). Don't feel particularly strongly about it though